keras/src/backend/numpy/nn.py

```diff
@@ -121,6 +121,13 @@ def celu(x, alpha=1.0):
     )
 
 
+def glu(x, axis=-1):
+    x = convert_to_tensor(x)
+    assert x.shape[axis] % 2 == 0, "axis size must be divisible by 2"
```

Comment:
Use a `ValueError` and specify "Received: x.shape={x.shape} with axis={axis}" in the error message